dataclass_csv/dataclass_reader.py

```diff
@@ -155,6 +164,11 @@ def _process_row(self, row):
                     values.append(transformed_value)
                     continue
 
+            if field.type is bool:
+                transformed_value = self._parse_bool_value(value)
```

Comment:
Here we would need to handle cases where the value cannot be parsed to boolean. If we are going to use the strtobool in the distutils module (https://docs.python.org/3.7/distutils/apiref.html?highlight=strtobool#distutils.util.strtobool) it will raise a ValueError exception.